### PR TITLE
Adding examples for yarn-label

### PR DESCRIPTION
**Pull request by sanjay990**

### CODE REVIEW

File: presto-docs/src/main/sphinx/installation/installation-yarn.rst [hunk 1]
```diff
@@ -643,10 +643,24 @@ Using YARN label
 This is an optional feature and is not required to run Presto in YARN.
 To guarantee that a certain set of nodes are reserved for deploying
 Presto or to configure a particular node for a component type we can
-make use of YARN label expressions.
+make use of YARN label expressions. To create and assign node to labels refer the below link.
+See http://docs.hortonworks.com/HDPDocuments/HDP2/HDP-2.3.0/bk\_yarn\_resource\_mgt/content/ch\_node\_labels.html
```
Comment (anusudarsan) [inline presto-docs/src/main/sphinx/installation/installation-yarn.rst]: You can keep either "refer the below link" or "See". not both

File: presto-docs/src/main/sphinx/installation/installation-yarn.rst [hunk 2]
```diff
@@ -643,10 +643,24 @@ Using YARN label
...
 
--  First assign the nodes/subset of nodes with appropriate labels. See
-   http://docs.hortonworks.com/HDPDocuments/HDP2/HDP-2.3.0/bk\_yarn\_resource\_mgt/content/ch\_node\_labels.html
+Let's say you have an 8 node cluster with
+3 master nodes - m1, m2, m3
+3 data nodes- d1, d2, d3
+2 edge nodes - e1, e2 (edge nodes are basically nodes with various client like Hbase, Hive, Oozie, etc)
+
+Without yarn lables, when we deploy presto-on-yarn the presto-coordinator/worker can be on any of the nodes d1,d2,d3.
```
Comment (anusudarsan) [inline presto-docs/src/main/sphinx/installation/installation-yarn.rst]: s/lables/labels

File: presto-docs/src/main/sphinx/installation/installation-yarn.rst [hunk 3]
```diff
@@ -643,10 +643,24 @@ Using YARN label
...
+In case, we want coordinator to be always on d1 and workers on d2 and d3, we can create 2 yarn labels.
+1. Coordinator
+2. Worker
+
+Assign coordinator to d1 and workers to d2 and d3. Now coordinator will always be running on d1.
+Note: YARN labels can only be assigned to nodes which have Node Manager service running on them.Generally data nodes are the nodes which have Node Manager service running.
```
Comment (anusudarsan) [inline presto-docs/src/main/sphinx/installation/installation-yarn.rst]: I would move this Note to line 654

Comment (anusudarsan) [inline presto-docs/src/main/sphinx/installation/installation-yarn.rst]: Formatting. Space after period in "them."

Comment (anusudarsan) [inline presto-docs/src/main/sphinx/installation/installation-yarn.rst]: s/Node Manager/YARN NodeManager

File: presto-docs/src/main/sphinx/installation/installation-yarn.rst [hunk 4]
```diff
@@ -643,10 +643,24 @@ Using YARN label
...
+3 master nodes - m1, m2, m3
+3 data nodes- d1, d2, d3
+2 edge nodes - e1, e2 (edge nodes are basically nodes with various client like Hbase, Hive, Oozie, etc)
+
+Without yarn lables, when we deploy presto-on-yarn the presto-coordinator/worker can be on any of the nodes d1,d2,d3.
+In case, we want coordinator to be always on d1 and workers on d2 and d3, we can create 2 yarn labels.
+1. Coordinator
+2. Worker
+
+Assign coordinator to d1 and workers to d2 and d3. Now coordinator will always be running on d1.
```
Comment (anusudarsan) [inline presto-docs/src/main/sphinx/installation/installation-yarn.rst]: I  think labels are case sensitive. So this should be

"Assign Coordinator to d1 and Worker to d2.. Now Presto coordinator will be always be running on d1".

File: presto-docs/src/main/sphinx/installation/installation-yarn.rst [hunk 5]
```diff
@@ -643,10 +643,24 @@ Using YARN label
...
+Note: YARN labels can only be assigned to nodes which have Node Manager service running on them.Generally data nodes are the nodes which have Node Manager service running.
+
+For assigning YARN labels to nodes, you can either use the Ambari Slider View( Create App-->Allocate Resource --> YARN Labels) or do it manually as given below.
```
Comment (anusudarsan) [inline presto-docs/src/main/sphinx/installation/installation-yarn.rst]: 1. Create App -> (add spaces) 
2. s/do it manually/add the label expression to resources.json if installing using Slider cli

File: presto-docs/src/main/sphinx/installation/installation-yarn.rst (hunk 1 above)
Comment (sanjay990) [inline presto-docs/src/main/sphinx/installation/installation-yarn.rst]: Modified - 060b866

File: presto-docs/src/main/sphinx/installation/installation-yarn.rst (hunk 2 above)
Comment (sanjay990) [inline presto-docs/src/main/sphinx/installation/installation-yarn.rst]: Modified - 060b866

**Comment by sanjay990** [comment]
Modified - 060b866

**Pull request closed without merging**